src/cantools/database/can/c_source.py

```diff
@@ -1606,7 +1657,7 @@ def _generate_helpers_kind(kinds, left_format, right_format):
     return helpers
 
 
-def _generate_helpers(kinds):
+def _generate_helpers(kinds: Tuple[Set[Tuple[str, int]], Set[Tuple[str, int]]]) -> str:
```

Comment:
what is the `Set[Tuple[str, int]]` here? introducing some type variable to make it more obvious would be nice:
```suggestion
HelperKind = Tuple[str, int]

def _generate_helpers(kinds: Tuple[Set[HelperKind], Set[HelperKind]) -> str:
```

(only if `Tuple[str, int]` is indeed a `HelperKind`)

Reply:
Done